hyper/http20/connection.py

```diff
@@ -122,17 +122,17 @@ def __init_state(self):
 
         # Values for the settings used on an HTTP/2 connection.
         self._settings = {
-            SettingsFrame.INITIAL_WINDOW_SIZE: 65535,
+            SettingsFrame.INITIAL_WINDOW_SIZE: 2**24,
         }
 
         # The socket used to send data.
         self._sock = None
 
         # The inbound and outbound flow control windows.
-        self._out_flow_control_window = 65535
+        self._out_flow_control_window = 2**24
```

Comment:
This *is* a flow control window, but it's not one we control: the remote side of the connection owns it. We can't just change it's value.